Content.Server/Atmos/Piping/Unary/EntitySystems/GasVentPumpSystem.cs

```diff
@@ -325,5 +336,25 @@ private void OnWeldChanged(EntityUid uid, GasVentPumpComponent component, ref We
         {
             UpdateState(uid, component);
         }
+        private void OnInteractUsing(EntityUid uid, GasVentPumpComponent component, InteractUsingEvent args)
+        {
+            if (args.Handled
+             || component.UnderPressureLockout == false
+             || !_toolSystem.HasQuality(args.Used, "Screwing")
+             || (!CompOrNull<TransformComponent>(uid)?.Anchored ?? false) // If component doesn't exist continue, else check for anchor, stops it from eating the deconstruction screwdrivering
```

Comment:
```suggestion
             || !Transform(uid).Anchored // check for anchor, stops it from eating the deconstruction screwdrivering
```
All entities have the TransformComponent, so no need to check for null. We can also use the shorthand for it.